Editor/EditorCore/PolyShapeEditor.cs

```diff
@@ -213,7 +213,7 @@ void RebuildPolyShapeMesh(bool vertexCountChanged = false)
             DrawPolyLine(polygon.m_Points);
 
             if(polygon.polyEditMode != PolyShape.PolyEditMode.Path)
-                polygon.CreateShapeFromPolygon();
+                polygon.mesh.CreateShapeFromPolygon(polygon.m_Points, polygon.extrude, polygon.flipNormals, SceneView.lastActiveSceneView.camera.transform.forward);
```

Comment:
just a suggestion but instead of passing all these here, you could modify CreateShapeFromPolygon() to take the cameraLookAt as an argument

Reply:
Indeed more elegant.